### ISSUE TITLE
Hyper Param/ Model optimization for leaning based reconstruction

### ISSUE THREAD

**Issue by PascalPolygon** [EarlyByrdTeam/ReconAlgorithm#1]
Things to tweak:
- Dim of RBF layer
- Various optimizers
- Various loss functions
- Various activation functions (on output layer)
- Experiment with various regularization techniques.
- More layers?

Metrics:
- Training score
- Misclassification rate on validation and test sets
- Time/epoch
- Time/inference
- Model complexity:
        - # of params
        - FLOP's
